### DISCUSSION TITLE
LFW layout

### DISCUSSION THREAD

**Discussion by devWonderkind** [virtual-tech-school/frytheweb#13] [Ideas]
@prakhartiwari0 
## How's it?
![image](https://github.com/virtual-tech-school/frytheweb/assets/110322061/b1614319-31f7-4ae8-85bc-6cd7ed954595)

**Comment by Vidya4sure** [comment]
@prakhartiwari0 

### mine also 
![image](https://github.com/virtual-tech-school/frytheweb/assets/123371804/6be16974-4e4e-4923-91d0-8cef4d006b96)
